src/feature-test/user_see_data_pending.test.js

```diff
@@ -0,0 +1,40 @@
+import '../setupTest.js'
+import { jsxEmptyExpression } from '@babel/types';
+import { SSL_OP_EPHEMERAL_RSA } from 'constants';
+
+describe('I.I.A.S', () => {
+  it('the user sees a message that data is being loaded when submit is pressed and before a response is returned', async () => {
+    try {
+      const browser = await puppeteer.launch({
+      });
+      const page = await browser.newPage();
+      await page.goto('http://localhost:3000/');
+      await page.waitForSelector('input[name="ing"]')
+      await page.type('input[name="ing"]', 'potatoes');
+      await page.click('#add-button');
+      await page.type('input[name="ing"]', 'sugar');
+      await page.click('#add-button');
+      await page.type('input[name="ing"]', 'milk');
+      await page.click('#add-button');
+      await page.setRequestInterception(true);
+      await page.on('request', interceptedRequest => {
+        if(interceptedRequest.resourceType(fetch)) {
+          interceptedRequest.abort();
+        }else{
+          interceptedRequest.continue();
+        }
+      });
+
+      await page.click('input[type="submit"]');
+
+      const pendingDataMessage = await page.evaluate(() => window.find("Please wait while we get the data from our partners....."));
+      expect(pendingDataMessage).toBe(true)
+      
+      await browser.close();
+
+    } catch (error){
+      console.log(error)
+    }
+    
+  });
+});
```

Comment:
Should have a new line at the end of the file.

Reply:
As above